distribution/ddf-common/src/main/resources/etc/jre.properties

```diff
@@ -0,0 +1,729 @@
+################################################################################
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+################################################################################
+
+#
+# Java platform package export properties.
+#
+
+# Standard package set.  Note that:
+#   - javax.transaction* is exported with a mandatory attribute
+jre-1.6= \
```

Comment:
❓ I know we only need this until a Karaf fix is released and we can upgrade to the next patch version, but would it make sense to clean this file up and remove the JRE1.6 and JRE1.7 sections?

Reply:
I had decided to preserve the same contents as the Karaf one except for commenting out the lines I need for the workaround, with comments to explain why they're commented out.